KoreBuild-dotnet/build/dotnet-install.sh

```diff
@@ -189,6 +189,9 @@ install_dotnet()
             say "Local Version: $localVersion"
 
             [ "$remoteHash" = "$localHash" ] && say "${green}You already have the latest version.${normal}" && return 0
+
+            dotnet_url="https://dotnetcli.blob.core.windows.net/dotnet/$CHANNEL/Binaries/$remoteVersion"
```

Comment:
In this case it's because KoreBuild.sh sends the -version argument to dotnet-install.sh with value 1.0.0.001244. That is, we're not moving to "Latest" every time you build.

The changes I'm proposing are simply doing what the script was supposed to do on a version change. Assume your computer has dotnet version 1.0.0.001244 but we change KoreBuild.sh to point to version 1.0.0.1333. The script in its current form will always download the wrong file and fail to upgrade you to the newly specified version.